lib/form/paragraph3.flow

```diff
@@ -197,20 +230,31 @@ renderParagraph(
 		})
 	));
 
+	size = make(WidthHeight(maxW, 0.0));
```

Comment:
size? where do you use it?

Reply:
Now nowhere, so I ought to comment this as it is done for Available2/Size2.